src/Middleware/Csrf.php

```diff
@@ -0,0 +1,95 @@
+<?php
+
+namespace Yiisoft\Yii\Web\Middleware;
+
+use Psr\Http\Message\ResponseFactoryInterface;
+use Psr\Http\Message\ResponseInterface;
+use Psr\Http\Message\ServerRequestInterface;
+use Psr\Http\Server\MiddlewareInterface;
+use Psr\Http\Server\RequestHandlerInterface;
+use Yiisoft\Router\Method;
+use Yiisoft\Security\Random;
+use Yiisoft\Security\TokenMasker;
+use Yiisoft\Yii\Web\Session\SessionInterface;
+
+final class Csrf implements MiddlewareInterface
+{
+    public const NAME = '_csrf';
```

Comment:
Are these constants useful outside of the class?

Reply:
I think yes, but besides this